### PR TITLE
Fix loading state regression in markup content

### PR DESCRIPTION
**Pull request by silverwind**
Fix regressions from https://github.com/go-gitea/gitea/pull/25219:

Math before and after:
Screenshot 2023-06-18 at 16 00 52
Screenshot 2023-06-18 at 16 03 44

Mermain before and after:
Screenshot 2023-06-18 at 15 58 56
Screenshot 2023-06-18 at 15 58 37

### CODE REVIEW

**Comment by silverwind** [comment]
@wxiaoguang There is one regression here by this change:

image

But I can't keep this `max-height` as it distorts this spinner:

Screenshot 2023-06-18 at 16 04 48

Any idea how to fix both? `aspect-ratio` seems to do nothing.

File: web_src/css/modules/animations.css [hunk 1]
```diff
@@ -18,7 +18,6 @@
   position: absolute;
   display: block;
   height: 4rem;
-  max-height: 50%;
```
Comment (wxiaoguang) [inline web_src/css/modules/animations.css:21]: Without max-height, does it work with when container's height < 4em?

**Comment by wxiaoguang** [comment]
> Any idea how to fix both? `aspect-ratio` seems to do nothing.

I guess there are a "width" style for it? Could you help to provide some reproducible code

**Comment by silverwind** [comment]
Seems setting both `max-width` and `max-height` seems to have fixed it:

Screenshot 2023-06-18 at 16 13 01
Screenshot 2023-06-18 at 16 14 18

File: web_src/css/modules/animations.css (hunk 1 above)
Comment (silverwind) [reply web_src/css/modules/animations.css:21]: Works now with `100%` in both directions it seems.

**Review by wxiaoguang — APPROVED** [review]

**Review by HesterG — APPROVED** [review]

**Comment by lunny** [comment]
workflow maybe wrong, test-e2e hasn't been triggered.

**Comment by silverwind** [comment]
This PR should only trigger frontend pipelines.

**Comment by silverwind** [comment]
e2e is running, looks fine now.

**Pull request merged**